packages/ui-webc/src/components/text-area/text-area.css

```diff
@@ -0,0 +1,21 @@
+:host {
+  display: inline-flex;
+}
+
+.flexbox{
+    display: inline-flex;
+    flex-direction: column;
+    padding: 0.5rem;
+}
+
+.textarea{
+    resize: none;
+    height: var(--spacing-24);
+    width: var(--spacing-50);
```

Comment:
I think we should keep the width and height unset so that the consumer can choose how to size the component. What we could do instead is expose the [rows attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/textarea#rows) as a prop and maybe set it to a higher default, say 3?